Added Kiali chart (#4939)
kyma-project/kyma: resources/kiali/README.md

# Kiali

[Kiali](http://kiali.io) is a spyglass for the istio service mesh and provides answers to the questions:
What microservices are part of my Istio service mesh? How are they connected? How are they performing?

## Introduction

This chart installs Kiali on a Kyma cluster.

## Prerequisites

> - The `monitoring` chart must be installed in order to have Kiali functional

Reviewer: ```suggestion

-  Install the Monitoring chart to make Kiali work.
```

> - The `jaeger` chart should be installed to have the Jaeger integration of Kiali functional

Reviewer: ```suggestion
- Install the Jaeger chart to integrate Jaeger with Kiali. 
```
Can we simply say `integrate` here or `to make Jaeger integration with Kiali work` sounds better? I would suggest the shorter version, but I don't want to change the meaning :)

> - The `logging` chart should be installed to have the Grafana integration of Kiali functional

Reviewer: ```suggestion
- Install the Logging chart to integrate Grafana with Kiali.
```
Also the same question as above.

## Installing the Chart

> To install the chart with the release name `kiali` in namespace kyma-system call:

Reviewer: ```suggestion
To install the chart with the release name `kiali` in the `kyma-system` Namespace, run:
```

    ```
    $ helm install kiali --name kiali --namespace kyma-system
    ```

> ## Uninstalling the Chart

Reviewer: ```suggestion
### Unistallation
```

> To uninstall/delete the `kiali` release but continue to track the release:

Reviewer: ```suggestion
To uninstall/delete the `kiali` release but continue tracking the release, run:
```
If we can still track the release maybe we can simply have `unistall` here without the `delete`?

    ```
    $ helm delete kiali
    ```

> To uninstall/delete the `kiali` release completely and make its name free for later use:

Reviewer: ```suggestion
To uninstall/delete the `kiali` release completely and make its name free for further use, run:
```
Maybe we can have `delete completely` without `unistall` here to distinguish it from the previous one?
